docs/blocking-schema.rst

```diff
@@ -0,0 +1,219 @@
+.. _blocking-schema:
+
+Blocking Schema
+===============
+Each blocking method has its own configuration and parameters to tune with. To make our API as generic
+as possible, we designed the blocking schema to specify the configuration of the blocking method including
+features to use in generating blocks and hyperparameters etc.
+
+Currently we support two blocking methods:
+
+* Probability signature
+
+* LSH based :math:`\lambda`-fold
+
+which are proposed by the following publication:
```

Comment:
```suggestion
which are proposed by the following publications:
```

Although I suggest we bring the publications and the name of the blocking methods back together. Say we supported lots of blocking methods.